script/ios_bootstrap.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import argparse
+import os
+import sys
+import pathlib
+
+from distutils.dir_util import copy_tree
+from lib.config import PLATFORM, enable_verbose_mode, is_verbose_mode
+from lib.util import execute_stdout, scoped_cwd
+
+SRC_DIR = os.path.normpath(os.path.join(os.path.dirname(__file__), '..', '..'))
```

Comment:
please remove `SRC_DIR` and replace all `os.path.join` things with `brave_chromium_utils.wspath()` calls.
https://github.com/brave/brave-core/blob/6fd9b197fadf864762ee98211b62a631aab4919d/script/brave_chromium_utils.py#L33-L38

Reply:
My PR is not fully rebased on master yet so I dont have `brave_chromium_utils` but I did use `import_util` for the time being, will replace with `brave_chromium_utils` on rebase